src/compiletest/compiletest.rs

```diff
@@ -22,7 +22,7 @@
 #![feature(unicode)]
 #![feature(env)]
 
-#![deny(warnings)]
+// #![deny(warnings)]
```

Comment:
I think those were existing warnings that only triggered in stage1 test builds. I'll see if its still an issue and either uncomment again or fix the problematic warnings.